### ISSUE TITLE
Dubious results with `brew file casklist`

### ISSUE THREAD

**Issue by rinopo** [rcmdnk/homebrew-file#18]
I see the following lines in `~/Caskfile`, which I find strange:

```
# Apps installed directly instead of by Cask in brew-cask
#cask install opera-mail #/Applications/Mail.app
#cask install serviio #/Applications/Utilities/Console.app
#cask install tinygrab #/Applications/Utilities/Grab.app
#cask install gpgtools #/Applications/Utilities/Keychain Access.app
```

The suggested casks don't seem to correspond with the preinstalled apps.
Is it homebrew-file or homebrew-cask being wrong?

Just to note, I have `export HOMEBREW_CASK_OPTS=--appdir=/Applications` in my `.bash_profile` (though I don't know if this has anything to do with the issue).

**Comment by rcmdnk** [comment]
It is because such `opera-mail` has words of "Mail.app" (in fact, it is "Opera Mail.app").
The `casklist` command checks all applications under "/Applications" and "$HOME/Applications",
and searches cask file.
(Note: currently such "--appdir" option is not valid for `casklist` command.)
In the last of the search sequence, script picks up a cask file in which the application name is written.
As a result, Mail.app was associated with opera-mail.

Although they are harmless, I put exceptions and 
they are ignored (listed in "no casks are available" in the bottom) in the new version (3.3.3).

**Comment by rinopo** [comment]
Thanks for the clarification!

I don't know if `brew file casklist` should respect `--appdir` (#20), if `casklist` is meant to scan the most possible places. But it's up to you.
